bvm/ballerina-config/pom.xml

```diff
@@ -45,6 +45,10 @@
             <groupId>org.ballerinalang</groupId>
             <artifactId>toml-parser</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
Cant' we generously borrow code of the StringEscapeUtils.unescapeJava and StringEscapeUtils.escapeJava. Rather than adding new dependency?